src/shared/messaging/port-util.js

```diff
@@ -35,7 +37,7 @@ export function isMessage(data) {
  * Return true if the data payload from a MessageEvent matches `message`.
  *
  * @param {any} data
- * @param {Message} message
+ * @param {Partial<Message>} message
```

Comment:
Isn't it too lenient to mark all the `Message` properties as optional? As suggested earlier, why not to test for `requestId` too?